source/libraries/ContractDeployer.ts

```diff
@@ -34,7 +34,7 @@ export class ContractDeployer {
         await this.uploadAllContracts();
         await this.initializeAllContracts();
         await this.whitelistTradingContracts();
-
+        
```

Comment:
Surprised your editor let you add this indent, I believe the editorconfig settings are "no whitespace on blank lines"?

Reply:
I'll add something